data-connector/package.json

```diff
@@ -16,7 +16,10 @@
     "graphql-tag": "^2.12.6",
     "inferable": "^0.30.48",
     "mysql2": "^3.11.5",
+    "node-fetch": "^3.3.2",
```

Comment:
I think this is not needed?

Reply:
Ahh yep, replaced with std lib fetch

https://github.com/inferablehq/inferable/pull/217/commits/72d8e8b7734a5604c618e70b659a2d444c63fb4a